content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -127,23 +147,48 @@ PRISMA_QUERY_ENGINE_BINARY=custom/my-query-engine-unix
 For Prisma CLI it allows you to define the query engine file to be used.  
 For Prisma Client, on build time (during `prisma generate`), it defines where the query engine file will be copied from into the Prisma Client. At run time (when using the generated Client) it can be used to define the specific query engine file to be used instead of the included one.
 
-### <inlinecode>PRISMA_INTROSPECTION_ENGINE_BINARY</inlinecode>
+Note: This can only have an effect if the engine type of CLI or Client are set to `binary`
+
+#### <inlinecode>PRISMA_QUERY_ENGINE_LIBRARY</inlinecode>
+
+`PRISMA_QUERY_ENGINE_LIBRARY` is used to set a custom location for your own query engine library.
+
+```terminal
+PRISMA_QUERY_ENGINE_LIBRARY=custom/my-query-engine-unix
+```
+
+For Prisma CLI it allows you to define the query engine file to be used.  
+For Prisma Client, on build time (during `prisma generate`), it defines where the query engine file will be copied from into the Prisma Client. At run time (when using the generated Client) it can be used to define the specific query engine file to be used instead of the included one.
```

Comment:
```suggestion
For Prisma Client:
 * At *build time* (during `prisma generate`): it defines where the query engine file will be copied from into the Prisma Client.
 * At *runtime* (when using the generated Client): it can be used to define the specific query engine file to be used instead of the included one.
```
Maybe too much, feel free to ignore

Reply:
I'll try this at the end - could look indeed too detailed with the formatting as list - but structurally this makes total sense.

Reply:
Decided against it as this is really just explanation people will read when they really need to understand the details, and putting this into a list would be "too shiny" and pull others people attention on this.